weaviate/collections/tenants.py

```diff
@@ -128,27 +138,71 @@ def get(self) -> Dict[str, Tenant]:
             `weaviate.UnexpectedStatusCodeError`
                 If Weaviate reports a non-OK status.
         """
-        path = "/schema/" + self.__name + "/tenants"
-        response = self.__connection.get(
-            path=path,
-            error_msg=f"Could not get collection tenants for {self.__name}",
-            status_codes=_ExpectedStatusCodes(
-                ok_in=200, error=f"Get collection tenants for {self.__name}"
-            ),
-        )
+        if self.__connection._weaviate_version.supports_tenants_get_grpc:
+            return self.__get_with_grpc()
+        else:
+            return self.__get_with_rest()
 
-        tenant_resp: List[Dict[str, Any]] = response.json()
-        return {tenant["name"]: Tenant(**tenant) for tenant in tenant_resp}
+    def get_by_names(self, names: Sequence[str]) -> Dict[str, Tenant]:
+        """Return named tenants currently associated with a collection in Weaviate.
+
+        If the tenant does not exist, it will not be included in the response.
+
+        If no names are provided, all tenants will be returned.
+
+        The collection must have been created with multi-tenancy enabled.
+
+        Arguments:
+            `names`
+                Optional list of tenant names to retrieve. If not provided, all tenants will be returned.
```

Comment:
in case that we leave tenant arguments in the other operations it should be added here, too

Reply:
I'm not sure how to align the nomenclature if we do that. Ideally, we'd rename `.get` to `.all` and then use `.get` with arguments but this isn't possible with BC. `get_by_names` seems off if the argument is `tenants`, WDYT?

Reply:
I think it is fine, I see it more as a curtesy to users that they don't have to do [tenant.name for tenant in tenants].